src/framework/ui/ikeynavigation.h

```diff
@@ -60,6 +69,13 @@ class IKeyNavigationSubSection : public IKeyNavigation
 public:
     virtual ~IKeyNavigationSubSection() = default;
 
+    enum class Direction {
+        Horizontal = 0,
+        Vertical,
+        Grid
```

Comment:
I think Layout is a better name for this enum because Grid isn't a direction

Reply:
I do not think it's a good idea.
This enum is about directions of navigation, not about location.
Perhaps the `Grid` can be renamed, for example to `Table`, or `Both`.

Reply:
I like the idea to rename Grid to Both

Reply:
renamed

Reply:
It's still "Grid" in keynavigationsubsection.h

If it's a direction of navigation, I'd propose to name it like "LeftToRight" or "FromLeftToRight". Because even in the horizontal direction you have 2 possible ways to change focus (from left to right, from right to left).

Reply:
> It's still "Grid" in keynavigationsubsection.h

Thanks, fixed